app/styles/app/modules/request/buttons.scss

```diff
@@ -0,0 +1,93 @@
+.repo-config .request-configs .request-configs-buttons {
+  position: relative;
+  text-align: right;
+
+  .request-configs-button-cancel {
+    display: none;
+  }
+}
+
+.request-configs .request-configs-buttons {
+  position: absolute;
+  top: 1px;
+  right: 0;
+}
+
+.request-configs-button {
+  display: inline-block;
+  vertical-align: bottom;
+  outline: none;
+  user-select: none;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
```

Comment:
You don't need to manually add vendor prefixes, Ember CLI's build pipeline includes Autoprefixer, which takes care of prefixes.

Reply:
oh, nice, i didn't know! done: https://github.com/travis-ci/travis-web/pull/2399/commits/642d68d96413b78b6400f26d4f4322404a04f186